config.py

```diff
@@ -22,6 +22,7 @@ def __init__(self):
 	S3_KEY = environ.get("S3_ACCESS_KEY", default=None)
 	S3_SECRET = environ.get("S3_SECRET_ACCESS_KEY", default=None)
 	S3_LOCATION = 'http://{}.s3.amazonaws.com/'.format(S3_BUCKET)
+	SERVER_NAME = 'fitzroyacademy.com'
```

Comment:
Are there any new local configuration requirements that need to be documented in the README?

https://flask.palletsprojects.com/en/1.1.x/config/#SERVER_NAME

Reply:
Yes, there are and were documented in README.